src/init.cpp

```diff
@@ -70,6 +70,7 @@
 #include <boost/algorithm/string/classification.hpp>
 #include <boost/algorithm/string/replace.hpp>
 #include <boost/algorithm/string/split.hpp>
+#include <boost/signals2/signal.hpp>
```

Comment:
Would this line be better in `src/ui_interface.h` instead? It seems to be where it's used. Build and tests green.

Reply:
it's used in init.cpp:366: `static boost::signals2::connection rpc_notify_block_change_connection;`

Reply:
d'oh, thanks -- not sure how I missed that.